lib/ts/recipe/emailpassword/components/themes/styles/styles.ts

```diff
@@ -181,6 +181,36 @@ export function getStyles(palette: NormalisedPalette): NormalisedDefaultStyles {
             margin: "0 auto",
         },
 
+        backButtonContainer: {
+            paddingBottom: "30px",
+        },
+
+        backButtonWithLeftArrow: {
+            display: "flex",
+            justifyContent: "center",
+            alignItems: "center",
+            padding: "0px",
+            cursor: "pointer",
+            lineHeight: "26px",
+            fontSize: "14px",
+            fontWeight: "300",
+            letterSpacing: "0.4px",
+            backgroundColor: "transparent",
+            border: "none",
+            margin: "0px auto",
+            fontFamily: "inherit",
```

Comment:
There are a few props that seem redundant.
- why do we need to specify `fontFamily: inherit` here?
- `margin` seems to be redundant as well.
- Also, doesn't this match the `secondaryText` style? (and `linkButton`?)

Reply:
Maybe reusing `secondaryText` is not the right call here, but please check existing styles to see if any fit here.

Reply:
Without `fontFamily: inherit`, the button element does not update it's font family as per the parent

For instance, the sign in button here does not have `fontFamily: inherit` so it does not use the fontFamily of it's parent
<img width="250" alt="Screenshot 2022-06-14 at 9 08 56 PM" src="https://user-images.githubusercontent.com/92283713/173618747-cc731b82-3b2c-4d88-90ee-5502851e6e65.png">

Reply:
`margin` is used to center the button horizontally

Reply:
1. I didn't know that about `button`s, thanks :)
2. The `display: flex` + related centering props confused me; it seemed either those or the margin should be redundant.

Reply:
It shouldn't matter now, `secondaryText secondaryLinkWithLeftArrow` styles are now used for the button which were already being used for a similar link in Passwordless form.

I have updated it